dist/modularity/access_check/check_sources.py

```diff
@@ -11,21 +11,29 @@
 
 # layercparse is a library written and maintained by the WiredTiger team.
 import layercparse as lcp
-import wt_defs
+
+def load_wt_defs(rootPath):
```

Comment:
I think, `load_wt_defs` should be a part of the library.

Reply:
Updated in [b59c9ce](https://github.com/wiredtiger/wiredtiger/pull/11335/commits/b59c9cec310c3af0787db6a4ec6596ee48b059f5)